### ISSUE TITLE
Calendar days and hours are not translated

### ISSUE THREAD

**Issue by horus68** [salesagility/SuiteCRM#330]
Please refactor calendar language file at modules\Calendar (lines 164 to 2016)
https://github.com/salesagility/SuiteCRM/blob/master/modules/Calendar/language/en_us.lang.php

The actual arrays needs to be changed a key-value style instead of the actual hardcoded language.
Else those lines should go into a separated file so it doesn't have to have the same array style as the SuiteCRM language files.

Actual arrays example:
```
$mod_list_strings = array(
	'dom_cal_weekdays'=>
		array(
			"Sun",
			"Mon",
			"Tue",
			"Wed",
			"Thu",
			"Fri",
			"Sat",
		),
	'dom_cal_weekdays_long'=>
		array(
			"Sunday",
			"Monday",
			"Tuesday",
			"Wednesday",
			"Thursday",
			"Friday",
			"Saturday",
		),
	'dom_cal_month'=>
		array(
			"",
			"Jan",
			"Feb",
			"Mar",
			"Apr",
			"May",
			"Jun",
			"Jul",
			"Aug",
			"Sep",
			"Oct",
			"Nov",
			"Dec",
		),
	'dom_cal_month_long'=>
		array(
			"",
			"January",
			"February",
			"March",
			"April",
			"May",
			"June",
			"July",
			"August",
			"September",
			"October",
			"November",
			"December",
		),
```

**Comment by horus68** [comment]
Even with those lines translated the calendar body is not translated. Same goes for time format that keep display in 11:pm and not 23:00
General settings and user settings all reset to same format. 
![calendario](https://cloud.githubusercontent.com/assets/3055443/21396504/6ffe4126-c798-11e6-9e0a-ab24f4cd5ecd.png)

Possible related issue: 
Module Body Border Text https://github.com/salesagility/SuiteCRM/issues/1772#issuecomment-237484867

**Comment by horus68** [comment]
@likhobory is your calendar columns translated (day and format hour)?

**Comment by likhobory** [comment]
I have the same problem.

**Comment by horus68** [comment]
@shogunpol re-confirmed as a bug.

**Comment by maladrill** [comment]
I confirm. There is the problem with resource Calendar as well - days of week and months cannot be translated
![resources_calendar](https://cloud.githubusercontent.com/assets/12087022/25522409/0921055e-2c03-11e7-9a86-17e603116628.PNG)

**Comment by horus68** [comment]
See this PR: Fix #2498 - Calendar Localization not working: days long and time slot by Szellszi · Pull Request #2983 · salesagility/SuiteCRM - https://github.com/salesagility/SuiteCRM/pull/2983

**Comment by likhobory** [comment]
It works.

**Comment by horus68** [comment]
@maladrill the resources calendar is now fixed in Suite 7.9.rc, even without this fix

**Comment by horus68** [comment]
@Dillon-Brown Calendar translation for days, months (and long hour format) are now OK (7.9.0 RC version- applied 2 PR fix: #2983 and yours #3565 )
On the Resources calendar it is not translated yet. Maybe its better to open a new Issue?

**Comment by Dillon-Brown** [comment]
@horus68 These are technically 3 separate issues, key-value, calendar localisation and resources calendar.  So feel free to create another one for just the resources calendar.
